tests/rdma/run.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python3
+"""
+==========================================================================
+run.py - script for test client for Valkey Over RDMA (Linux only)
+--------------------------------------------------------------------------
+Copyright (C) 2024  zhenwei pi <pizhenwei@bytedance.com>
+
+This work is licensed under BSD 3-Clause, License 1 of the COPYING file in
+the top-level directory.
+==========================================================================
+"""
+import os
+import subprocess
+import netifaces
+import time
+
+def build_program():
+    cmd = "gcc tests/rdma/rdma-test.c -o tests/rdma/tmp/valkey-rdma-test -lrdmacm -libverbs -lpthread -g"
```

Comment:
We support other compilers too but looking at how we compile the C files under test/modules/ I see it's hardcoded to gcc for Linux too. Thus, I think hardcoded to gcc is OK for now.